cve_bin_tool/available_fix/__init__.py

```diff
@@ -52,3 +57,9 @@ def get_backport_supported_distros():
             ]
     supported_distros.append("local")
     return supported_distros
+
+
+def get_available_fix_supported_distros():
+    """Generates a list for --available-fix option in distro-distro_codename fashion"""
+
+    return get_backport_supported_distros()
```

Comment:
This function only returns `get_backport_supported_distros` now but will be used with redhat support (Also using `get_backport_supported_distros` with `available_fix` seems annoying in my mind )